Library/Homebrew/global.rb

```diff
@@ -157,6 +158,8 @@ def auditing?
 require "tap"
 require "tap_constants"
 
+Tap.lib_directories.each { |lib_dir| LoadPath << lib_dir }
```

Comment:
This would allow any tap to open any Homebrew class/module and modify the code within.

Reply:
That’s correct; is the same thing not already possible today from inside a formula/cask body?

Reply:
@claui No, that is run in a sandboxed, forked process.